src/commands.cpp

```diff
@@ -1301,12 +1289,12 @@ void CommandPutNodes::procresult()
             }
         }
 #endif
-        client->app->putnodes_result((!e && empty) ? API_ENOENT : e, type, nn);
+        client->app->putnodes_result((!errorDetails && empty) ? API_ENOENT : errorDetails.getError(), type, nn);
```

Comment:
Here you can simply use a casting to `error`, and avoid the extra getter ;)

Reply:
Yes, it's good